packages/backend/src/peripherals/database/shared/BaseRepository.ts

```diff
@@ -188,12 +230,18 @@ export abstract class BaseRepository {
     )
   }
 
+  protected wrapSet<T extends SetMethod>(method: T): T {
+    return this.wrap(method, (count) =>
+      this.logger.info({ method: method.name, count }),
```

Comment:
why does set method have count argument?

Reply:
This returns how many row were updated as a result of this function call. Maybe the result there should be `success: boolean` instead of this count. WDYT?

Reply:
I guess it's fine, approved.